### DISCUSSION TITLE
How to create a dashed underline for textbox text

### DISCUSSION THREAD

**Discussion by YogitaGurnani** [fabricjs/fabric.js#8531] [Q&A]
Hi All,

I am trying to achieve the dashed underline text as mentioned in the below screenshot. The text can be in multiple lines and all the lines should have the dashed underline.

<img width="76" alt="image" src="https://user-images.githubusercontent.com/50671651/208887640-8c0da562-42a2-4044-b116-78d35b9b060d.png">

Does anyone know how to do this?

**Comment by ShaMan123** [comment]
Look at overriding `Text#_renderTextDecoration`
